# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1465932694 14400
#      Tue Jun 14 15:31:34 2016 -0400
# Node ID a46c49f0d52e347772f7c886cccdf8e90334761e
# Parent  7338be52b8926a288ee00adeb766e930de95bddb
* nproc-wrapper.c: Set default value for query (bug #48139)

diff --git a/liboctave/wrappers/nproc-wrapper.c b/liboctave/wrappers/nproc-wrapper.c
--- a/liboctave/wrappers/nproc-wrapper.c
+++ b/liboctave/wrappers/nproc-wrapper.c
@@ -31,17 +31,17 @@ along with Octave; see the file COPYING.
 
 #include "nproc.h"
 
 #include "nproc-wrapper.h"
 
 unsigned long int
 octave_num_processors_wrapper (enum octave_nproc_query octave_query)
 {
-  enum nproc_query query;
+  enum nproc_query query = NPROC_CURRENT;
 
   switch (octave_query)
     {
     case OCTAVE_NPROC_ALL:
       query = NPROC_ALL;
       break;
 
     case OCTAVE_NPROC_CURRENT:
